### PR TITLE
Fix: Splash Screen not consistently working on all devices 

### PR DESCRIPTION
**Pull request by aditya-bhawsar**
As suggested removed splash screen api implementation and added a uniform background theme fi

https://user-images.githubusercontent.com/50818905/178348878-c21d7e60-0e2a-4dd4-8e06-2bc38f9c9474.mp4

https://user-images.githubusercontent.com/50818905/178349162-180d6cc0-1a5e-44d2-af30-4b64c148efe1.mp4

### CODE REVIEW

**Comment by Anmol92verma** [comment]
Can we have a quick gif to see the impl both light and dark mode, also in dark mode is it matching the color with original app ?

**Comment by aditya-bhawsar** [comment]
added gifs and corrected the colour

**Review by shubhamsinghshubham777 — APPROVED** [review]

File: app/src/main/java/dev/baseio/discordjetpackcompose/MainActivity.kt [hunk 1]
```diff
@@ -29,9 +28,11 @@ class MainActivity : ComponentActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        installSplashScreen()
-        /**For Making the App FullScreen*/
+
+        setTheme(R.style.Theme_DiscordJetpackCompose)
```
Comment (shubhamsinghshubham777) [inline app/src/main/java/dev/baseio/discordjetpackcompose/MainActivity.kt:32]: @aditya-bhawsar Can we move this above super.onCreate?

File: app/src/main/res/values-night/themes.xml [hunk 2]
```diff
@@ -5,13 +5,8 @@
...
-        <!-- Use windowSplashScreenAnimatedIcon to add either a drawable or an
-             animated drawable. One of these is required. -->
-        <item name="windowSplashScreenAnimatedIcon">@drawable/dark_app_logo</item>
-
-        <!-- Set the theme of the Activity that directly follows your splash screen. -->
-        <!-- Required -->
-        <item name="postSplashScreenTheme">@style/Theme.DiscordJetpackCompose</item>
+    <style name="SplashTheme" parent="Theme.DiscordJetpackCompose">
+        <item name="android:windowBackground">@drawable/splash_image_dark</item>
+        <item name="android:statusBarColor">@color/black</item>
```
Comment (shubhamsinghshubham777) [inline app/src/main/res/values-night/themes.xml:10]: @aditya-bhawsar Can we use transparent color for both status and navigation bars just like the original app?

**Review by shubhamsinghshubham777 — CHANGES_REQUESTED** [review]

File: app/src/main/java/dev/baseio/discordjetpackcompose/MainActivity.kt (hunk 1 above)
Comment (aditya-bhawsar) [reply app/src/main/java/dev/baseio/discordjetpackcompose/MainActivity.kt:30]: done

File: app/src/main/res/values-night/themes.xml (hunk 2 above)
Comment (aditya-bhawsar) [reply app/src/main/res/values-night/themes.xml]: done

**Review by shubhamsinghshubham777 — APPROVED** [review]

**Pull request merged**